meta-resin-common/recipes-bsp/u-boot/patches/resin-specific-env-configuration.patch

```diff
@@ -31,12 +27,18 @@ index 0000000..2ff0fa5
 + *     resin_kernel_load_addr - kernel load address as we use the same
 + *                              to load the env file
 + *     resin_root_part        - existing in the env file to import (optional)
++ *     resin_boot_from_root   - if existent, load boot files from root (optional)
++ *     resin_boot_counter     - if existent the root partition counter file
++ *                              will be used to set correct root part. Set to 1
++ *                              to get high count or 2 to get low count (optional)
```

Comment:
Any usecase for having both low and high count?

Reply:
Yes. This is from @petrosagg's mail regarding implementing boot counter and thus selecting new/old images: